server/src/index.ts

```diff
@@ -13,82 +14,119 @@ if (!fs.existsSync(uploadDir)) {
 }
 
 const ENDPOINTS = {
-  deleteEmployee: "/api/delete-employee",
-  addEmployee: "/api/add-employee",
-  getEmployees: "/api/employees",
+  employee: "/api/employee",
+  employees: "/api/employees",
 };
 
-const validateEmployeeObject = (obj: EmployeeI): boolean => {
+const validateEmployeeObject = (obj: IEmployee): boolean => {
   return !!obj?.name && !!obj?.lastName && !!obj?.occupation && !!obj?.age;
 };
 
-const getAllFiles = (): Array<EmployeeI> => {
-  const data: Array<EmployeeI> = [];
-  fs.readdirSync(uploadDir).forEach(file => {
-    const content = fs.readFileSync(`${uploadDir}${file}`);
-    data.push(JSON.parse(content.toString()) as EmployeeI);
-  });
+const parseAllFiles = async (): Promise<Array<IEmployee>> => {
+  const data: Array<IEmployee> = [];
+  const files = await readdir(uploadDir);
+  for (const file of files) {
+    const buffer = await readFile(`${uploadDir}/${file}`);
+    const content = JSON.parse(buffer.toString());
+    data.push(content);
+  }
   return data;
 };
 
-const deleteEmployee = (fileName: string): boolean => {
-  let deleted = false;
+const deleteEmployee = async (id: string): Promise<string | null> => {
```

Comment:
Why the function that is above is named 'parseAllFiles' but this function is 'deleteEmployee', it's done not in one style

Reply:
I changed deleteEmployee and addEmployee functions to deleteFile and addFile  for consistency